tools/format.py

```diff
@@ -15,6 +14,8 @@
 parser.add_argument("--gn", help="only run gn format", action="store_true")
 parser.add_argument("--cc", help="only run clang format", action="store_true")
 
+clang_format_path = os.path.join(third_party_path, "depot_tools",
+                                 "clang-format")
```

Comment:
Could we directly call `clang_format.py` with `sys.executable` here instead of the wrapper script?

Reply:
It is possible, but the way it was done in #2943 did not work for me (on windows).
Also, if bypassing wrapper scripts is a goal, then IMO we should just call the clang_format binary directly.
The clang_format.py wrapper creates more problems than it solves.

Reply:
Oh, but without it I've seen strange build errors on appveyor like: https://ci.appveyor.com/project/deno/deno/builds/27388163#L329. But unfortunately I can't test it by myself on Windows, so it's  quite hard to debug for me.

Reply:
IMHO calling the clang-format binary directly seems to be a good option for me, because otherwise after the gn root move to `core/libdeno` we would need to either set `CHROMIUM_BUILDTOOLS_PATH` or maintain 2 `buildtools` symlinks, see: https://github.com/denoland/deno/pull/2943#discussion_r324442881